pkg/skuba/actions/cluster/init/init.go

```diff
@@ -87,6 +87,7 @@ func Init(initConfiguration InitConfiguration) error {
 			return errors.Wrap(err, "unable to render template")
 		}
 		f.WriteString(str)
+		f.Chmod(0600)
```

Comment:
@mjura this would make all scaffold files have the 600 permissions. I currently can not think of an issue with this. and seemed to work just fine.